src/process/process.cr

```diff
@@ -48,15 +48,39 @@ class Process
       ret = LibC.kill(pid.to_i32, signal.value)
       raise Errno.new(ret) if ret < 0
     end
-    0
+    nil
+  end
+
+  def self.kill(signal : Signal, *procs : Process)
```

Comment:
Do we really need to receive an enumeration of processes here? As said before, I'd remove this method and leave `Process#kill`.